lib/matplotlib/axes/_base.py

```diff
@@ -325,7 +325,7 @@ def _makefill(self, x, y, kw, kwargs):
         seg = mpatches.Polygon(np.hstack((x[:, np.newaxis],
                                           y[:, np.newaxis])),
                                facecolor=facecolor,
-                               fill=True,
+                               fill=kwargs.get('fill', True),
```

Comment:
histogram steps (for one) go through this path, but sometimes pass `fill=False`.  If that gets ignored, than the patch will have neither fill or stroke in the new scheme here.